Common/include/basic_types/ad_structure.hpp

```diff
@@ -547,17 +594,15 @@ FORCEINLINE void SetPreaccOut() {}
 template <class T, class... Ts, su2enable_if<std::is_same<T, su2double>::value> = 0>
 FORCEINLINE void SetPreaccOut(T& data, Ts&&... moreData) {
   if (!PreaccActive) return;
-  if (IsIdentifierActive(data)) PreaccHelper.addOutput(data);
+  PreaccHelper.addOutput(data);
   SetPreaccOut(moreData...);
 }
 
 template <class T>
 FORCEINLINE void SetPreaccOut(T&& data, const int size) {
   if (PreaccActive) {
     for (int i = 0; i < size; i++) {
-      if (IsIdentifierActive(data[i])) {
-        PreaccHelper.addOutput(data[i]);
-      }
+      PreaccHelper.addOutput(data[i]);
```

Comment:
Any efficiency impact from this change? Or codi is already doing something equivalent internally?

Reply:
Yes it is. It's no longer needed :)